sources/c_api/legacy_hw_path/qpl_hw_get_size.h

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ ******************************************************************************/
+
+#ifndef QPL_HW_GET_SIZE_H
+#define QPL_HW_GET_SIZE_H
+
+// includes extracted from sources/c_api/legacy_hw_path/qpl_hw_get_size.c
+#include "own_defs.h"
+#include "hardware_state.h"
+
+
+#endif
```

Comment:
Unclear on the value of such changes if we're just moving out a couple of header files, so I'd rather avoid it.

Reply:
got it! how do you feel about limiting the additional header files to those which are necessary to cover the new declarations?

Reply:
Do you mean similar to changes in `sources/c_api/compression_operations/qpl_deflate_slow.h`?

Reply:
yes, that's one example -- there are a few more in the third commit that are similar.